src/main/webapp/app/lecture/manage/lecture-units/management/lecture-unit-management.component.ts

```diff
@@ -107,6 +114,7 @@ export class LectureUnitManagementComponent implements OnInit, OnDestroy {
     transcriptionStatus: Record<number, TranscriptionStatus> = {};
     playlistUrls: Record<number, string> = {};
     isTranscriptionLoading: Record<number, boolean> = {};
+    isUploadingPdfs = signal<boolean>(false);
```

Comment:
In all three components we have this `isUploadingPdfs` signal, but its only written to and never read. You probably want to disable the pdf drop zone when it's true or show a loading spinner or something